daemon/src/maker.rs

```diff
@@ -294,15 +290,19 @@ async fn main() -> Result<()> {
         Poll::Ready(Some(message))
     });
 
-    tasks.add(incoming_connection_addr.attach_stream(listener_stream));
+    tasks.add(
+        maker_actor_system
+            .inc_conn_addr
+            .clone()
+            .attach_stream(listener_stream),
```

Comment:
```suggestion
        maker_actor_system.listen_on(listener)
```

I think we could hide a few more details here, that should allow us to make all fields of `MakerActorSystem` private.